### ISSUE TITLE
Change icon resolution of apk file?

### ISSUE THREAD

**Issue by sylvio2000** [koying/SPMC#254]
On FireTV there is the problem, that the icons of sideloaded apps look really bad, because Amazon uses only 96x96 pixel instead of 512x512.

There is a workaround to change the icon inside the apk file to a resolution of 512x512. Then everything looks fine. But now everytime the app gets updated the old low-res icon is back.
Here is more info on how to change the icon:
http://www.aftvnews.com/how-to-use-custom-icons-for-sideloaded-apps-like-kodi-on-the-amazon-fire-tv/

Is it possible for you to find a solution for this problem?

**Comment by koying** [comment]
Not really. Overriding the 96x96 icon would probably lead to issues on all non-AFTV devices.
I might do a "special" AFTV build, but seems a lot of work/hack for just a nice icon...
